documentation/status-logging.md

```diff
@@ -0,0 +1,55 @@
+# Status logging
+
+TK-status can periodically log status data to a file in JSON format.
+
+To take advantage of this you must do two things:
+
+1. Enable logging through you TK app's configuration. For example, to log every 2 seconds, your configuration might look like this:
```

Comment:
The units in the example here seem inconsistent.  The description says "to log every 2 seconds" but the HOCON snippet has `interval-minutes: 5`.  Should we change the text to say "to log every 5 minutes"?